README.md

```diff
@@ -2,7 +2,14 @@
 
 This repo contains a large file uploader app and a theme for Liferay's Zendesk Help Center.
 
+## Large File Uploader App
+
+A simple app that shows up in the agent view that displays a link to an external large file uploader page that allows the user to upload a file greater than the Zendesk attachment file size limit.
```

Comment:
Can we update this text to `A simple app that appears in the agent view; it displays a link to a page with an external large file uploader allowing users to upload a file larger than the Zendesk attachment file size limit.`

Reply:
:ok_hand: